srcpkgs/Sequeler/template

```diff
@@ -0,0 +1,15 @@
+# Template file for 'Sequeler'
+pkgname=Sequeler
+version=0.6.0
+revision=1
+wrksrc="sequeler-${version}"
+build_style=meson
+hostmakedepends="pkg-config glib-devel vala"
```

Comment:
There is an xbps-trigger to run glib-compile-resources, the build should not do it manually when it detects DESTDIR. Looking at `meson/post_install.py`, glib-devel should not be necessary.

Reply:
well, it fails to build without it
```
FileNotFoundError: [Errno 2] No such file or directory: 'glib-compile-resources': 'glib-compile-resources'
```

Reply:
Oh I see the meson.build uses that command. It should be okay to keep then.